examples/postSubmissionApplication/data/lawfulDevelopmentCertificate/proposed.ts

```diff
@@ -0,0 +1,111 @@
+import {PostSubmissionApplication} from '../../../../types/schemas/postSubmissionApplication';
+import {lawfulDevelopmentCertificateProposedPrototype} from '../../../prototypeApplication/data/lawfulDevelopmentCertificate/proposed';
+
+const version = process.env['VERSION'] || '@next';
+
+/**
+ * This example is awaiting determination still
+ * there is no consultation period for LDC applications so no consultation section
+ */
+export const lawfulDevelopmentCertificateProposedPostSubmission: PostSubmissionApplication =
+  {
+    applicationType: 'ldc.proposed',
+    data: {
+      application: {
+        reference: 'ABC-123-XYZ',
+        stage: 'assessment',
+        status: 'undetermined',
+      },
+      localPlanningAuthority: {
+        commentsAcceptedUntilDecision: false,
+      },
+      submission: {
+        submittedAt: '2025-02-05T11:33:56.009Z',
+      },
+      validation: {
+        receivedAt: '2025-02-05T11:33:56.009Z',
+        validatedAt: '2025-02-05T12:33:56.009Z',
+        isValid: true,
+      },
+      caseOfficer: {
+        name: 'Casey Officer',
+      },
+    },
+    submission: lawfulDevelopmentCertificateProposedPrototype,
+    metadata: {
+      publishedAt: '2025-01-10T13:00:00.288Z',
+      organisation: 'BKM',
+      id: '8da51c5b-a2a0-4386-a15d-29d66f9c121c',
+      source: 'PlanX',
```

Comment:
Is the `source` actually still 'PlanX' here?

Reply:
at some points yes but not always! I've updated the schema to extend from the base metadata now too so theres more options!